src/PrizePool.sol

```diff
@@ -170,7 +174,7 @@ contract PrizePool {
         uint96 _reserveShares,
         UD2x18 _claimExpansionThreshold,
         SD1x18 _smoothing
-    ) {
+    ) Ownable(msg.sender) {
```

Comment:
Darn....can't do this because there are more than 11 parameters to the constructor.....

Maybe we could hard-code the draw period seconds to 24 hours....